### PR TITLE
add template

### PR DESCRIPTION
**Pull request by kiennt175**
## Related Tickets:
- [#29070](https://edu-redmine.sun-asterisk.vn/issues/29070)

## Notes Impacted Areas:
- N/A.

## Attachments:
add_template

### CODE REVIEW

**Comment by kiennt175** [comment]
ready

File: resources/views/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+            <div class="col-lg-6 col-xl-6">
+                <div class="banner_text">
+                    <div class="banner_text_iner">
+                        <h5>Every child yearns to learn</h5>
+                        <h1>Making Your Childs
+                            World Better</h1>
+                        <p>Replenish seasons may male hath fruit beast were seas saw you arrie said man beast whales
+                            his void unto last session for bite. Set have great you'll male grass yielding yielding
+                            man</p>
+                        <a href="" class="btn_1">View Course </a>
```
Comment (huyenttm-0901) [inline resources/views/index.blade.php]: i18n

**Review by huyenttm-0901 — COMMENTED** [review]
i18n (tat)

**Comment by kiennt175** [comment]
ready

**Pull request merged**